lib/elastomer/client/scroller.rb

```diff
@@ -128,19 +128,22 @@ class Scroller
       #
       def initialize( client, query, opts = {} )
         @client = client
-        @query  = query
 
-        @index       = opts.fetch(:index, nil)
-        @type        = opts.fetch(:type, nil)
-        @scroll      = opts.fetch(:scroll, '5m')
-        @size        = opts.fetch(:size, 50)
-        @search_type = opts.fetch(:search_type, nil)
+        default_opts = {
+          :index => nil,
+          :type => nil,
+          :scroll => '5m',
+          :size => 50,
+          :body => query,
+        }
```

Comment:
I'd put these in a constant above the method like so:

```ruby
DEFAULT_OPTS = {
  :index  => nil,
  :type   => nil,
  :scroll => '5m',
  :size   => 50
}.freeze
```

I'd probably move the `:body` merge somewhere else, since it's already an ivar.